### ISSUE TITLE
NVDA user is unable to change the month and year in the 'Calendar' while using calendar short cut keys

### ISSUE THREAD

**Issue by nagamanasa123** [nvaccess/nvda#12443]
Test Environment:
OS: WIN10 20H2(OS Build 19042.685)
Browser: NewEdge
URL: Claim Offer - Get 3 More Months of Microsoft 365 (epaydev.com)
Screen Readers used: NVDA 2020.4

Repro Steps:
Step 1: Enable Screen reader(NVDA) and open the above application in latest Edgebrowser.
Step 2: 'Claim Offer' page will be displayed.
Step 3: Navigate to 'Get started' section->  'Accept our Terms and Conditions' checkbox(select) -> Submit button -> Status Check section -> Enter valid code in all 5 segments text fields ->  Continue button -> Contact details section -> Enter valid email in Email fields -> Continue buttons.
Step 4: Purchase details section is displayed.
Step 5: Navigate to the 'Purchase Date' edit field and expand the calendar.
Step 6: Verify Screen reader is able to change the using calendar shortcut keys or not.
 
Actual Result:
NVDA user is unable to change the month and year in the 'Calendar' window while using calendar short cut keys
Observation: Upon invoking the 'Calendar' shortcut keys PageUp/PageDown, Home/End keys user is unable to change the month and years and focus is moving away from the 'Calendar' window.

Expected Result:
User should be able to navigate through the calendar using the calendar shortcut keys (PageUp, PageDown,Home,End) while navigating using NVDA and also user should also access the >,< buttons in the calendar.
Ensure that while navigate in the calendar window focus should not move out of the 'Calendar' window.

Note : Screen Readers (NVDA) is not narrating the ‘March 2021’ information present in calender, while navigating using down arrow key.When NVDA is on ‘March 2021’ button just narrating as ‘button Show month selection panel’.

https://user-images.githubusercontent.com/49755113/118979180-d503e500-b995-11eb-80a7-7019cb3bd149.mp4

https://user-images.githubusercontent.com/49755113/118979240-e3ea9780-b995-11eb-8ce6-424462df7630.mp4

**Comment by Adriani90** [comment]
@nagamanasa123 iut seems you use browse mode and that's why shortcuts don't work. If you want to use the native callendar shortcuts, you have to enable focus mode in NVDA by pressing nvda+space bar for example. Or by pressing enter on a button in which case NVDA should automatically enable focus mode when the callendar window opens. Automatic change to focus mode should be enabled in NVDA browse mode settings for this to work properly.
Closing as works for me.
If you are still having this issue, please comment and we can reopen.
